kfs-core/src/main/resources/edu/arizona/kfs/gl/businessobject/datadictionary/AccountingLineLiteBo.xml

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<beans xmlns="http://www.springframework.org/schema/beans" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:p="http://www.springframework.org/schema/p" xsi:schemaLocation="http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans-2.0.xsd">
+    <bean id="AccountingLineLiteBo" parent="BusinessObjectEntry">
+        <property name="businessObjectClass" value="edu.arizona.kfs.gl.businessobject.lite.AccountingLineLiteBo" />
+        <property name="objectLabel" value="AccountingLineLiteBo" />
+        <property name="attributes">
+            <list>
+                <ref bean="AccountingLineLiteBo-documentNumber" />
+                <ref bean="AccountingLineLiteBo-sequenceNumber" />
+                <ref bean="AccountingLineLiteBo-financialDocumentLineTypeCode" />
+                <ref bean="AccountingLineLiteBo-objectId" />
+                <ref bean="AccountingLineLiteBo-versionNumber" />
+                <ref bean="AccountingLineLiteBo-chartOfAccountsCode" />
+                <ref bean="AccountingLineLiteBo-accountNumber" />
+                <ref bean="AccountingLineLiteBo-postingYear" />
+                <ref bean="AccountingLineLiteBo-financialObjectCode" />
+                <ref bean="AccountingLineLiteBo-balanceTypeCode" />
+                <ref bean="AccountingLineLiteBo-amount" />
+                <ref bean="AccountingLineLiteBo-subAccountNumber" />
+                <ref bean="AccountingLineLiteBo-financialSubObjectCode" />
+                <ref bean="AccountingLineLiteBo-projectCode" />
+                <ref bean="AccountingLineLiteBo-organizationReferenceId" />
+                <ref bean="AccountingLineLiteBo-referenceOriginCode" />
+                <ref bean="AccountingLineLiteBo-referenceNumber" />
+                <ref bean="AccountingLineLiteBo-referenceTypeCode" />
+                <ref bean="AccountingLineLiteBo-overrideCode" />
+                <ref bean="AccountingLineLiteBo-financialDocumentLineDescription" />
+                <ref bean="AccountingLineLiteBo-debitCreditCode" />
+                <ref bean="AccountingLineLiteBo-encumbranceUpdateCode" />
+            </list>
+        </property>
+    </bean>
+
+<!--     Attribute Definitions -->
+    <bean id="AccountingLineLiteBo-documentNumber"                      parent="SourceAccountingLine-documentNumber"                                                            />

```

Comment:
What's going on with the attribute formatting here? It's pushed out to 179 characters for all these beans.